index.js

```diff
@@ -22,7 +22,8 @@ module.exports = function (npmLifecycle, options, cb) {
 var spawnWrapper = function (options, scriptFile) {
   return function (cb) {
     printScript(scriptFile)
-    var child = spawn(scriptFile, options.spawn)
+    var userArgs = options.userArgs || process.argv.slice(2)
```

Comment:
To avoid unnecessary branching and to keep the process-aware bits separated, please parse argv in the cli.js file and pass in the args using the userArgs option (which will probably eliminate this variable)